README.md

```diff
@@ -5,34 +5,35 @@
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright (c) 2015, Joyent, Inc.
 -->
 
+
 # SmartDataCenter
 
 SmartDataCenter (SDC) is an open-source cloud management platform, optimized
 to deliver next generation, container-based, service-oriented infrastructure
-across one or more datacenters.  With an emphasis on ease of installation
-and operation, SDC is proven at scale:  it is the software that runs
-the [Joyent public cloud](https://www.joyent.com/products/compute-service)
+across one or more data centers. With an emphasis on ease of installation
+and operation, SDC is proven at scale: it is the software that runs
+the [Joyent public cloud](https://www.joyent.com/public-cloud)
 and powers
 numerous [private clouds](https://www.joyent.com/products/private-cloud)
 at some of the world's largest companies.
 
-This repo provides documentation for the overall SDC project and pointers to the
-other repositories that make up a complete SDC deployment. See the [repository
-list](./docs/developer-guide/repos.md).
+This repository provides documentation for the overall SDC project and
+pointers to the other repositories that make up a complete SDC deployment.
+See the [repository list](./docs/developer-guide/repos.md).
 
-To report bugs or request features, submit issues to the [joyent/sdc
-project](https://github.com/joyent/sdc/issues).
+Report bugs or request features here on GitHub, [joyent/sdc/issues](https://github.com/joyent/sdc/issues).
```

Comment:
Went with "Report bugs and request features using [GitHub Issues](https://github.com/joyent/sdc/issues)." for now.